tests/models/beit/test_modeling_beit.py

```diff
@@ -387,6 +409,251 @@ def test_model_from_pretrained(self):
         model = BeitModel.from_pretrained(model_name)
         self.assertIsNotNone(model)
 
+    @parameterized.expand([("float16",), ("bfloat16",), ("float32",)])
+    @require_torch_sdpa
+    def test_eager_matches_sdpa_inference(self, torch_dtype: str):
```

Comment:
Could you please add some comment what was modified with respect to the common test